set_up/to-do-app-playbook.yml

```diff
@@ -0,0 +1,80 @@
+- name: Install to-do app on new web servers
+  hosts: webservers
+  remote_user: ec2-user
+
+- hosts: all
+  vars_prompt:
+  - name: boardKey
+    prompt: enter boardKey value
+
+
+  tasks:
+  - name: Install the latest versions of git and python
+    become: yes
+    yum:
+      name: git
+      state: latest
+      name: python3
+      state: latest
```

Comment:
You've duplicated the "name" and "state" keys.

This is like trying to define this dictionary in python:
```python
yum = { 'name': 'git', 'state': 'latest', 'name': 'python3', 'state': 'latest' }
```

It doesn't make any sense - what is `yum['name']` supposed to be equal to? It can only have one value.

To install two tools with yum in a single task, you need to give "name" a list of package names
```yml
name:
  - git
  - python3
```